lib/matplotlib/axes/_base.py

```diff
@@ -1286,26 +1286,68 @@ def apply_aspect(self, position=None):
                 self.set_xbound((x0, x1))
 
     def axis(self, *v, **kwargs):
-        """
-        Convenience method for manipulating the x and y view limits
-        and the aspect ratio of the plot. For details, see
-        :func:`~matplotlib.pyplot.axis`.
+        """Set axis properties.
+
+        If no positional or kwargs are given, returns the current
+        axes limits are returned with no side effects.
+
+        If a positional argument is passed the keyword arguments are ignored.
+
+        If no positional argument is passed, the keyword arguments are used.
+
+        Parameters
+        ----------
+        v : list of float or {'on', 'off', 'equal', 'tight', 'scaled',\
+         'normal', 'auto', 'image', 'square'}
```

Comment:
Sorry, I was referring to the actual padding in the table. It's not consistent with the other tables, but that's just an aesthetic thing.